script/dump_templates

```diff
@@ -174,7 +179,7 @@ my $client = OpenQA::Client->new(apikey => $options{'apikey'}, apisecret => $opt
 
 my %result;
 
-for my $table (qw/Machines TestSuites Products JobTemplates/) {
+for my $table (@all_tables) {
```

Comment:
Taking that we (used to) keep it in git, preserving an order sounds good to me. But we can gain that by calling sort over %keys :)